src/helpers.h

```diff
@@ -6,6 +6,7 @@
 #include "liteclient.h"
 
 void generate_correlation_id(Uptane::Target& t);
```

Comment:
I guess if we add return type to the existing function then it won't break existing clients because they don't expect any return value anyway.

Reply:
Key change was the input parameter, changed from `Uptane::Target` to `TufTarget`. The method is only called inside aklite daemon main loop, and the original version is not used after the changes in this PR. I've now removed it, keeping only the new method.

Reply:
OK. I would just make the param const: `const TufTarget&`.